deploy/mutatingwebhookconfiguration.yaml

```diff
@@ -17,6 +17,6 @@ webhooks:
       resources: [ "builds" ]
     failurePolicy: Fail
     matchPolicy: Exact
-    timeoutSeconds: 30s
+    timeoutSeconds: 30
     sideEffects: None
-    admissionReviewVersions: v1beta1
\ No newline at end of file
+    admissionReviewVersions: [v1beta1]
```

Comment:
```
admissionReviewVersions: ["v1beta1"] 
```
`admissionReviewVersions` type is array (string) - from the [doc](https://docs.openshift.com/container-platform/4.8/rest_api/extension_apis/mutatingwebhookconfiguration-admissionregistration-k8s-io-v1.html)

Reply:
Yes, this is an array of strings.